classes/02class/exercises/c04-iam01/gpavelar/README.md

```diff
@@ -0,0 +1,75 @@
+# c04-iam01
+
+## Command Execution Output
+
+- [cli_commands.txt](cli_commands.txt)
+- [s3-policy-document.json](s3-policy-document.json)
+
+## Errors
+
+I got some errors while doing the exercies, some mistakes about files and policies.
```

Comment:
I am always pleased to see this section in your PRs. It is optional but by writing it you are absorbing more. Well done.